smaily-for-contact-form-7.php

```diff
@@ -42,14 +42,24 @@
 // Required to use functions is_plugin_active and deactivate_plugins.
 require_once ABSPATH . 'wp-admin/includes/plugin.php';
 
-define( 'SMAILY_FOR_CF7_VERSION', '1.0.1' );
+define( 'SMAILY_FOR_CF7_VERSION', '1.0.2' );
 
 /**
  * The core plugin class that is used to define
  * admin-specific hook and public-facing site hooks.
  */
 require plugin_dir_path( __FILE__ ) . 'includes/class-smaily-for-cf7.php';
 
+/**
+ * Load translations files.
+ *
+ * @since 1.0.0
+ */
+function smaily_for_cf7_load_textdomain() {
+	load_plugin_textdomain( 'smaily-for-contact-form-7', false, basename( dirname( __FILE__ ) ) . '/languages/' );
```

Comment:
Why did you decide to use `basename( dirname( __FILE__ ) )` on determining plugin's path (instead `plugin_dir_path( __FILE__ )`)?